[new-product] Veeam Backup & Replication

adrianwells opened the pull request:
An attempt to add a product page for Veeam Backup & Replication

adrianwells committed:
Create veeam-backupandreplication

Add new product page for Veeam Backup & Replication

marcwrobel commented on products/veeam-backupandreplication:
```diff
@@ -0,0 +1,111 @@
+category: app
+tags: veeam
+iconSlug: veeam
+permalink: /veeam-backupandreplication
```
```suggestion
permalink: /veeam-backup-and-replication
```

WDYT of this (for both the permalink and the filename) ?

marcwrobel commented on products/veeam-backupandreplication:
```diff
@@ -0,0 +1,111 @@
+iconSlug: veeam
+permalink: /veeam-backupandreplication
+alternate_urls:
+-   /veeambackupandreplication
```
```suggestion
alternate_urls:
-   /veeam-backup
```

Not sure `veeambackupandreplication` will be useful to anybody. Maybe `/veeam-backup` if this name is used as a contraction for the product ?

marcwrobel commented on products/veeam-backupandreplication:
Odd, the deploy preview is broken: https://deploy-preview-3522--endoflife-date.netlify.app/veeam-backupandreplication.

Will have a look at it when I have more time.

marcwrobel commented on products/veeam-backupandreplication:
```diff
@@ -0,0 +1,111 @@
+    link: null
+
+-   releaseCycle: "9.0"
+    releaseDate: 2016-01-01
```
Both 9.0 and 9.5 share the same dates ?

marcwrobel commented on products/veeam-backupandreplication:
```diff
@@ -0,0 +1,111 @@
+    link: null
+
+-   releaseCycle: "6.0"
+    releaseDate: 2011-01-01
```
Should be 2011-08-22 according to https://www.veeam.com/kb2680.

Other release dates are wrong too (6.5 for example), could you re-check versions against https://www.veeam.com/kb2680 ?

marcwrobel commented on products/veeam-backupandreplication:
```diff
@@ -0,0 +1,111 @@
+-   /veeambackupandreplication
+releasePolicyLink: https://www.veeam.com/product-lifecycle.html
+activeSupportColumn: End of Fix
+releaseColumn: false
```
https://www.veeam.com/kb2680 contains a lot of patch versions with their date, maybe we could try to follow these (we could write a script in [release-data](https://github.com/endoflife-date/release-data/) to automatically apply those patch updates).

adrianwells replied:
Sounds like a good future task.

marcwrobel commented on products/veeam-backupandreplication:
```diff
@@ -0,0 +1,111 @@
+
+> [Veeam Backup & Replication](https://helpcenter.veeam.com/docs/backup/qsg_vsphere/about_vbr.html?ver=120) is a data protection and disaster recovery solution for virtual, physical and cloud environments.
+
+## [End of Fix](https://www.veeam.com/product-lifecycle.html)
```
No need to use link in titles: the presence of a `releasePolicyLink` will trigger the generation of short paragraph for it.

For example for Ruby this is:

> More information is available on the [Ruby website](https://www.ruby-lang.org/en/downloads/releases/).

marcwrobel reviewed (changes requested):
Thanks a lot for this contribution @adrianwells ! I left you a few notes and suggestions, do not hesitate if you need help.

adrianwells committed:
Update and rename veeam-backupandreplication to veeam-backup-and-replication

Update following review feedback from https://github.com/endoflife-date/endoflife.date/pull/3522

marcwrobel commented on products/veeam-backup-and-replication:
The file should be called `products/veeam-backup-and-replication.md` (it is missing the `.md` suffix)

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+releases:
+-   releaseCycle: "12.0"
+    releaseDate: 2023-01-30
+    support: true
```
```suggestion
    support: true # will be 13.0 GA date
```

According to https://www.veeam.com/product-lifecycle.html.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+releaseDateColumn: true
+eolColumn: End of support
+
+releases:
```
```suggestion
# Release dates (RTM) can be found on https://www.veeam.com/kb2680.
# EOL dates can be found on https://www.veeam.com/product-lifecycle.html.
releases:
```

To help maintainers.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+eolColumn: End of support
+
+releases:
+-   releaseCycle: "12.0"
```
```suggestion
-   releaseCycle: "12"
```

It seems they do not use the minor version anymore (source https://www.veeam.com/kb2680).

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    eol: 2026-02-01
+    link: "https://www.veeam.com/kb4420"
+
+-   releaseCycle: "11.0a"
```
```suggestion
-   releaseCycle: "11a"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    eol: 2024-02-01
+    link: "https://www.veeam.com/kb4245"
+
+-   releaseCycle: "11.0"
```
```suggestion
-   releaseCycle: "11"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+---
+
+> [Veeam Backup & Replication](https://helpcenter.veeam.com/docs/backup/qsg_vsphere/about_vbr.html?ver=120) is a data protection and disaster recovery solution for virtual, physical and cloud environments.
+
```
Maybe something should be added about the difference between RTM and GA ?

marcwrobel replied:
From what I can see on https://www.veeam.com/product-lifecycle.html it looks like latest versions reached end of fix when the next major was released and end of support one year later. Maybe we should document this ? It would be even better if this was confirmed somewhere on the Veeam website.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    support: 2023-02-01
+    eol: 2024-02-01
+    link: "https://www.veeam.com/kb4126"
+
```
Release 10a, documented on https://www.veeam.com/kb2680, is missing.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    eol: 2024-02-01
+    link: "https://www.veeam.com/kb4126"
+
+-   releaseCycle: "10.0"
```
```suggestion
-   releaseCycle: "10"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+
+releases:
+-   releaseCycle: "12.0"
+    releaseDate: 2023-01-30
```
Should we use RTM of GA dates ? I would feel more natural to me to use the GA date, but I am not a Veeam user so I don't now.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2020-02-04
+    support: 2021-02-01
+    eol: 2023-02-01
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: "https://www.veeam.com/kb3161"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2016-11-16
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: "https://www.veeam.com/kb2970"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2016-01-12
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: "https://www.veeam.com/kb2147"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2014-11-06
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: "https://www.veeam.com/kb2068"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2013-08-20
+    support: false    
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: "https://www.veeam.com/kb1891"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2012-10-09
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: "https://www.veeam.com/kb1751"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2012-06-04
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: "https://www.veeam.com/kb1671"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2011-08-22
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: "https://www.veeam.com/kb1442"
```

According to https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2010-08-30
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: null
```

Nothing on https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    releaseDate: 2009-10-29
+    support: false
+    eol: true
+    link: "https://www.veeam.com/kb2680"
```
```suggestion
    link: null
```

Nothing on https://www.veeam.com/kb2680.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "4.0"
+    releaseDate: 2009-10-29
```
Where can I confirm this date?

adrianwells replied:
[This](https://en.wikipedia.org/wiki/Veeam_Backup_%26_Replication) has some information.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    link: "https://www.veeam.com/kb2680"
+
+-   releaseCycle: "3.0"
+    releaseDate: 2009-01-01
```
Where can I confirm this date?

adrianwells replied:
[This](https://en.wikipedia.org/wiki/Veeam_Backup_%26_Replication) has some information.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    link: null
+
+-   releaseCycle: "2.0"
+    releaseDate: 2008-07-30
```
Where can I confirm this date?

adrianwells replied:
[This](https://en.wikipedia.org/wiki/Veeam_Backup_%26_Replication) has some information.

marcwrobel commented on products/veeam-backup-and-replication:
```diff
@@ -0,0 +1,123 @@
+    link: null
+
+-   releaseCycle: "1.0"
+    releaseDate: 2008-02-26
```
Where can I confirm this date?

adrianwells replied:
[This](https://en.wikipedia.org/wiki/Veeam_Backup_%26_Replication) has some information.

marcwrobel reviewed (changes requested):
Thanks a lot @adrianwells. I looked closely at the dates and noticed a few thing.

adrianwells committed:
Update and rename veeam-backup-and-replication to veeam-backup-and-replication.md

Additional updates based on review https://github.com/endoflife-date/endoflife.date/pull/3522

marcwrobel commented on products/veeam-backup-and-replication.md:
```diff
@@ -12,84 +12,92 @@ releaseColumn: false
--   releaseCycle: "12.0"
-    releaseDate: 2023-01-30
-    support: true
+-   releaseCycle: "12 GA"
```
It's not a good idea to indicate GA/RTM in the `releaseCycle`. `releaseCycle` are used as the name of the version in the API so it must be predictable.

I propose to remove GA/RTM from the `releaseCycle` and just say in the description that the GA date is used as the release date, except for 5.0 because it was not provided. WDYT @adrianwells ?

adrianwells replied:
OK. Makes sense. Updated.

marcwrobel commented on products/veeam-backup-and-replication.md:
```diff
@@ -12,84 +12,92 @@ releaseColumn: false
 releaseDateColumn: true
 eolColumn: End of support
 
+# Release dates (RTM) can be found on https://www.veeam.com/kb2680.
```
```suggestion
# Release dates (GA, or RTM as a fallback) can be found on https://www.veeam.com/kb2680.
```

marcwrobel commented on products/veeam-backup-and-replication.md:
```diff
@@ -12,84 +12,92 @@ releaseColumn: false
     support: false    
     eol: true
-    link: "https://www.veeam.com/kb2680"
+    link: "http://www.veeam.com/kb1891"
```
```suggestion
    link: "https://www.veeam.com/kb1891"
```

marcwrobel commented on products/veeam-backup-and-replication.md:
```diff
@@ -12,84 +12,92 @@ releaseColumn: false
     support: false
     eol: true
-    link: "https://www.veeam.com/kb2680"
+    link: "http://www.veeam.com/kb1442"
```
```suggestion
    link: "https://www.veeam.com/kb1442"
```

marcwrobel committed:
Update products/veeam-backup-and-replication.md

marcwrobel committed:
Update products/veeam-backup-and-replication.md

adrianwells committed:
Adrianwells patch 1 (#5)

* Update branch (#4)

* Update products/veeam-backup-and-replication.md

* Update products/veeam-backup-and-replication.md

---------

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

* Update veeam-backup-and-replication.md

Updates based on feedback

---------

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adrianwells committed:
Update products/veeam-backup-and-replication.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

marcwrobel committed:
Set support / EOL for 9.5

marcwrobel committed:
Update description

Based on https://en.wikipedia.org/wiki/Veeam_Backup_%26_Replication.

marcwrobel committed:
update description

Try to improve and simplify the description, while keeping the most important information.

marcwrobel committed:
Remove Xa versions
